### DISCUSSION TITLE
[Question] GTK 4

### DISCUSSION THREAD

**Discussion by ghost** [rafaelmardojai/firefox-gnome-theme#348] [Q&A] [answered]
Hello, 
Does you have any intent to actually add a "version" for the future libawaita/GTK4 version who change the appearance (more flat than this one).

I only ask to know if i wait an update or if i have to try to fix it myself (or work with other user on other repo)

Thanks

**Comment by rafaelmardojai** [comment, marked as answer]
There's a WIP `libawaita` branch that has been progressed by contributors. I plan to finish it a few weeks after Fedora 36 is released.
